frontend/src/entities/configuration/configuration-history-table.tsx

```diff
@@ -34,20 +35,23 @@ export function ConfigurationHistoryTable () {
     });
 
     useMemo(() => {
-        dispatch(listConfigurations({configScope: 'global', numVersions: 10}));
+        dispatch(listConfigurations({configScope: 'global', numVersions: 999999}));
```

Comment:
In reality I'm sure this works fine, but a more precise `numVersions` value would be passing in the versionId of the app config returned by `getConfiguration` (plus 1) since there will be up to `versionId + 1` versions.

Reply:
That's a significantly more clever solution. Even though I have all approvers, I want to change it to that. It makes more sense.